[BUG] BOM version uses v-prefix (v8.5.0) in Gradle Module Metadata causing issues with Maven repository managers

### JobRunr Version

v8.5.0

### JDK Version

OpenJDK Runtime Environment Temurin-21.0.10+7 (build 21.0.10+7-LTS)

### Your SQL / NoSQL database

Postgres

### What happened?

jobrunr version: 8.5.0

## Summary
The Gradle Module Metadata (.module file) for `jobrunr-spring-boot-3-starter` 
declares the BOM dependency with a `v` prefix version:

  "group": "org.jobrunr",
  "module": "jobrunr-bom",
  "version": { "requires": "v8.5.0" }

While the main artifact uses version `8.5.0` (no prefix), the BOM uses `v8.5.0`.
This inconsistency causes failures with Maven repository managers (e.g. ProGet, 
potentially others) that do not correctly handle v-prefixed Maven versions.

## Details
The file (https://repo1.maven.org/maven2/org/jobrunr/jobrunr/8.5.0/jobrunr-8.5.0.module) declares the version as v8.5.0 at line 31 for jobrunr-bom. This is not valid according to the Maven specification : https://docs.oracle.com/middleware/1212/core/MAVEN/maven_version.htm.

The build.gradle in root folder correctly declares the version in line 58
`version = System.getenv('CI_COMMIT_TAG')?.replace("v", "") ?: "v1.0.0-SNAPSHOT".replace("v", "")` 

however the platform/build.gradle line 7 seems to not replace the char "v"
`version = System.getenv('CI_COMMIT_TAG') ?: "v1.0.0-SNAPSHOT"`

This is causing issues with artifact repository ProGet. 
Link to ProGet discussion on this : https://forums.inedo.com/topic/5575/proget-unexpected-redirect-when-accessing-maven-package-with-non-standard-version-starting-with-a-character

I will try to create a patch for this.

### How to reproduce?

## Steps to Reproduce
1. Add `org.jobrunr:jobrunr-spring-boot-3-starter:8.5.0` to a Gradle build
2. Route dependencies through a Maven repository manager (e.g. ProGet)
3. Build fails — repository manager cannot resolve `jobrunr-bom:v8.5.0`

NB: if you need help with setting up Proget, please let me know.

### Relevant log output

```shell
2026-03-10T02:04:54.302+1100 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 
2026-03-10T02:04:54.303+1100 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] FAILURE: Build failed with an exception.
2026-03-10T02:04:54.303+1100 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 
2026-03-10T02:04:54.303+1100 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] * Where:
2026-03-10T02:04:54.303+1100 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] Build file '/Users/davisdavid/work/bitbucket_repos/swift/build.gradle' line: 187
2026-03-10T02:04:54.303+1100 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 
2026-03-10T02:04:54.303+1100 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] * What went wrong:
2026-03-10T02:04:54.303+1100 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] A problem occurred evaluating root project 'cooee-services'.
2026-03-10T02:04:54.304+1100 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] > Could not resolve all files for configuration ':testRuntimeClasspath'.
2026-03-10T02:04:54.304+1100 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]    > Could not resolve org.jobrunr:jobrunr-bom:v8.5.0.
2026-03-10T02:04:54.304+1100 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]      Required by:
2026-03-10T02:04:54.304+1100 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]          root project : > org.jobrunr:jobrunr-spring-boot-3-starter:8.5.0
2026-03-10T02:04:54.304+1100 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]          root project : > org.jobrunr:jobrunr-spring-boot-3-starter:8.5.0 > org.jobrunr:jobrunr:8.5.0
2026-03-10T02:04:54.304+1100 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]       > Could not resolve org.jobrunr:jobrunr-bom:v8.5.0.
2026-03-10T02:04:54.304+1100 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]          > Could not get resource 'https://revtrac-store.xrsc.com/maven2/cooee-group/org/jobrunr/jobrunr-bom/v8.5.0/jobrunr-bom-v8.5.0.pom'.
2026-03-10T02:04:54.304+1100 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]             > Redirecting from secure protocol to insecure protocol, without explicit opt-in, is unsupported. 'https://revtrac-store.xrsc.com/maven2/cooee-group/' is redirecting to 'http://revtrac-store.xrsc.com/maven2/cooee-group/org/jobrunr/jobrunr-bom/v8.5.0/jobrunr-bom-v8.5.0.pom/'. 
2026-03-10T02:04:54.304+1100 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]       > Could not resolve org.jobrunr:jobrunr-bom:v8.5.0.
2026-03-10T02:04:54.304+1100 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]          > Could not get resource 'https://revtrac-store.xrsc.com/maven2/cooee-releases/org/jobrunr/jobrunr-bom/v8.5.0/jobrunr-bom-v8.5.0.pom'.
2026-03-10T02:04:54.304+1100 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]             > Redirecting from secure protocol to insecure protocol, without explicit opt-in, is unsupported. 'https://revtrac-store.xrsc.com/maven2/cooee-releases/' is redirecting to 'http://revtrac-store.xrsc.com/maven2/cooee-releases/org/jobrunr/jobrunr-bom/v8.5.0/jobrunr-bom-v8.5.0.pom/'. 
2026-03-10T02:04:54.304+1100 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 
2026-03-10T02:04:54.304+1100 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] * Try:
2026-03-10T02:04:54.304+1100 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] > Switch Maven repository 'maven(https://revtrac-store.xrsc.com/maven2/cooee-group/)' to redirect to a secure protocol (like HTTPS) or allow insecure protocols. 
2026-03-10T02:04:54.304+1100 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] > For more information, please refer to https://docs.gradle.org/8.13/dsl/org.gradle.api.artifacts.repositories.UrlArtifactRepository.html#org.gradle.api.artifacts.repositories.UrlArtifactRepository:allowInsecureProtocol in the Gradle documentation.
2026-03-10T02:04:54.304+1100 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] > Switch Maven repository 'maven2(https://revtrac-store.xrsc.com/maven2/cooee-releases/)' to redirect to a secure protocol (like HTTPS) or allow insecure protocols. 
2026-03-10T02:04:54.304+1100 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] > Run with --stacktrace option to get the stack trace.
2026-03-10T02:04:54.304+1100 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] > Run with --scan to get full insights.
2026-03-10T02:04:54.304+1100 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] > Get more help at https://help.gradle.org.
2026-03-10T02:04:54.304+1100 [ERROR] [org.gradle.internal.buildevents.BuildResultLogger] 
2026-03-10T02:04:54.304+1100 [ERROR] [org.gradle.internal.buildevents.BuildResultLogger] BUILD FAILED in 6s
```

--- a/platform/build.gradle
+++ b/platform/build.gradle
@@ -4,8 +4,6 @@ plugins {
     id 'signing'
 }
 
-version = System.getenv('CI_COMMIT_TAG')?.replace("v", "") ?: "v1.0.0-SNAPSHOT".replace("v", "")
-
 dependencies {
     constraints {
         api 'org.slf4j:slf4j-api:2.0.17'

